fastlane/Snapfile

```diff
@@ -3,6 +3,7 @@ devices([
   'iPhone 6 Plus',
   'iPhone 5',
   'iPhone 4s',
+  'iPad Air',
```

Comment:
All of the iPads use the same screen resolution so we don't need to take any others.